crates/python/pyproject.toml

```diff
@@ -5,6 +5,7 @@ build-backend = "maturin"
 [project]
 name = "cryo_python"
 requires-python = ">=3.7"
+dependencies = ["maturin>=1.1,<2.0","pandas==2.1.3","polars==0.19.19","pyarrow==14.0.1"]
```

Comment:
why include maturin or pandas?

Reply:
Maturin should not have been there, but pandas is needed in "crates/python/python/cryo/_collect.py"

Reply:
Added and tested!